plat/mediatek/common/mtk_sip_svc.h

```diff
@@ -42,19 +42,30 @@
 #define MTK_SIP_SVC_VERSION_MAJOR	0x0
 #define MTK_SIP_SVC_VERSION_MINOR	0x1
 
+#define SMC_AARCH64_BIT		0x40000000
+
 /* Number of Mediatek SiP Calls implemented */
-#define MTK_SIP_NUM_CALLS		4
+#define MTK_SIP_NUM_CALLS		5
```

Comment:
Seems to be conditional on MTK_SIP_KERNEL_BOOT_ENABLE and the implementation of mediatek_plat_sip_handler()